LondonBritishLibrary/orient/BLorient658.xml

```diff
@@ -32,22 +33,55 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 236</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
-                        <summary/>
-                        
-                        
-                        
-                    </msContents>
-                    
-                    
-                    
-                    
-                    
-                    
+<msItem xml:id="ms_i1">
+<locus from="1r"/>
+<title xml:lang="gez" type="complete" ref="LIT2375Synaxa#FirstHalf">The first part of the Synaxarium</title>
```

Comment:
This is the "second recension", we can use the ID LIT4035SenkessarDL (the xml:ids after # can remain the same)